src/components/register/RegisterUser.js

```diff
@@ -17,7 +17,7 @@ export class RegisterUser extends Component {
   componentWillReceiveProps(nextProps) {
     if (nextProps.registerUserSuccess === true) {
       const { history } = this.props;
-      history.push('/');
+      history.push('/login');
```

Comment:
@MuhanguziDavid nice work, is it possible to take the user to the `dashboard` and not the `login` since registration also sets a token?

Reply:
@mendozabree , I do not think that this is necessary since the PO stated that a user should be directed to login after signup.

Reply:
Oh! Okay, thanks for letting me know